docs/builds.rst

```diff
@@ -53,17 +54,23 @@ Read the Docs cannot create a PDF version of your documentation with the *Mkdocs
 Understanding what's going on
 -----------------------------
 
-Understanding how Read the Docs builds your project will help you with debugging the problems you have with the site. It should also allow you to take advantage of certain things that happen during the build process.
+Understanding how Read the Docs builds your project will help you with debugging the problems you have with the site.
+It should also allow you to take advantage of certain things that happen during the build process.
 
-The first step of the process is that we check out your code from the repository you have given us. If the code is already checked out, we update the copy to the branch that you have specified in your projects configuration.
+The first step of the process is that we check out your code from the repository you have given us.
+If the code is already checked out, we update the copy to the branch that you have specified in your projects configuration.
 
 Then we build the proper backend code for the type of documentation you've selected.
 
-If you have the *Install Project* option enabled, we will run ``setup.py install`` on your package, installing it into a virtual environment. You can also define additional packages to install with the *Requirements File* option.
+If you have the *Install Project* option enabled, we will run ``setup.py install`` on your package, installing it into a virtual environment.
+You can also define additional packages to install with the *Requirements File* option.
 
-When we build your documentation, we run `sphinx-build -b html . _build/html`, where `html` would be replaced with the correct backend. We also create man pages and pdf's automatically based on your project.
+When we build your documentation, we run `sphinx-build -b html . _build/html`,
+where `html` would be replaced with the correct backend.
+We also create pdf's and ePub's automatically based on your project.
```

Comment:
The rest of the lines are the same, they were just too long. Except here, here we have had `man pages`, but we don't build man pages anymore.

Reply:
Don't we need double ` for the command and `html` here?

Reply:
Yeah